samples/azure_files/azure_files.r

```diff
@@ -0,0 +1,44 @@
+# =================
+# ===== Setup =====
+# =================
+
+# install packages
+library(devtools)
+install_github("azure/razurebatch")
```

Comment:
i thought we made razurebatch a dependency of doazureparllel so that we wouldn't have to do this?

Reply:
true :)